tests/build_test.sh

```diff
@@ -119,4 +120,11 @@ function test_build_info()
   rm .config
 }
 
+function test_kernel_build_pkg()
+{
+  output=$(kernel_build 'TEST_MODE' '--pkg' | head -1)
+  expected_result="make -j$PARALLEL_CORES ARCH=x86_64  bindeb-pkg"
+  assertEquals "($LINENO)" "$expected_result" "$output"
+}
```

Comment:
I suppose that you'll add more tests, right? I mean, you added a lot of new functions and new behavior... we need to capture them in the test.